homeassistant/components/xbox/sensor.py

```diff
@@ -0,0 +1,84 @@
+"""Xbox friends binary sensors."""
+from functools import partial
+from typing import Dict, List
+
+from homeassistant.const import STATE_UNKNOWN
+from homeassistant.core import callback
+from homeassistant.helpers.entity_registry import (
+    async_get_registry as async_get_entity_registry,
+)
+from homeassistant.helpers.typing import HomeAssistantType
+
+from . import XboxUpdateCoordinator
+from .base_sensor import XboxBaseSensorEntity
+from .const import DOMAIN
+
+SENSOR_ATTRIBUTES = ["status", "gamer_score", "account_tier", "gold_tenure"]
+
+
+async def async_setup_entry(hass: HomeAssistantType, config_entry, async_add_entities):
+    """Set up Xbox Live friends."""
+    coordinator: XboxUpdateCoordinator = hass.data[DOMAIN][config_entry.entry_id][
+        "coordinator"
+    ]
+
+    update_friends = partial(async_update_friends, coordinator, {}, async_add_entities)
+
+    unsub = coordinator.async_add_listener(update_friends)
+    hass.data[DOMAIN][config_entry.entry_id]["sensor_unsub"] = unsub
+    update_friends()
+
+
+class XboxSensorEntity(XboxBaseSensorEntity):
+    """Representation of a Xbox presence state."""
+
+    @property
+    def state(self):
+        """Return the state of the requested attribute."""
+        if not self.coordinator.last_update_success:
+            return STATE_UNKNOWN
```

Comment:
We use `None` to represent unknown state. It will be converted in the base entity class.
```suggestion
            return None
```